docs/operating-scylla/nodetool-commands/tasks-get-tree.rst

```diff
@@ -0,0 +1,35 @@
+Nodetool tasks-get-tree
+=======================
+**tasks-get-tree** - Gets the statuses of a task manager task and all its descendants.
+The statuses are listed in BFS order. If the task was finished it is unregistered.
+
+Syntax
+-------
+.. code-block:: console
+
+   nodetool tasks-get-tree <task_id>
```

Comment:
`nodetool task tree`, and make the parameter optional so we can list all tasks as a tree.